#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_preferences.dpatch by  <fred@candle>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: use Debian sensible-browser

@DPATCH@
diff -urNad wireshark-0.10.13~/configure.in wireshark-0.10.13/configure.in
--- wireshark-0.10.13~/configure.in	2005-12-08 12:47:00.000000000 +0100
+++ wireshark-0.10.13/configure.in	2005-12-08 12:53:33.000000000 +0100
@@ -38,12 +38,12 @@
 	#
 	AC_MSG_ERROR(I couldn't find pod2html; make sure it's installed and in your path)
 fi
-AC_PATH_PROG(HTML_VIEWER, htmlview)
+AC_PATH_PROG(HTML_VIEWER, sensible-browser)
 if test "x$HTML_VIEWER" = x
 then
-	AC_DEFINE_UNQUOTED(HTML_VIEWER, "mozilla", [HTML viewer, e.g. mozilla])
+	AC_DEFINE_UNQUOTED(HTML_VIEWER, "sensible-browser", [HTML viewer, e.g. sensible-browser])
 else
-	AC_DEFINE_UNQUOTED(HTML_VIEWER, "htmlview", [HTML viewer, e.g. mozilla])
+	AC_DEFINE_UNQUOTED(HTML_VIEWER, "sensible-browser", [HTML viewer, e.g. mozilla])
 fi
 
 AC_PATH_PROG(LEX, flex)
#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_preferences.dpatch by  <fpeters@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: use known working monospace font and Debian sensible-browser

@DPATCH@
diff -urNad --exclude=CVS --exclude=.svn ./epan/prefs.c /tmp/dpep-work.b0Uwke/ethereal-0.10.11/epan/prefs.c
--- ./epan/prefs.c	2005-07-17 18:36:36.000000000 +0200
+++ /tmp/dpep-work.b0Uwke/ethereal-0.10.11/epan/prefs.c	2005-07-17 18:38:45.000000000 +0200
@@ -1026,7 +1026,7 @@
      */
     prefs.gui_font_name1 = g_strdup("-misc-fixed-medium-r-semicondensed-*-*-120-*-*-*-*-iso8859-1");
     /* XXX- is this the correct default font name for GTK2 none win32? */
-    prefs.gui_font_name2 = g_strdup("fixed medium 12");
+    prefs.gui_font_name2 = g_strdup("monospace 10");
 #endif
     prefs.gui_marked_fg.pixel        =     65535;
     prefs.gui_marked_fg.red          =     65535;
@@ -1046,7 +1046,7 @@
     prefs.gui_fileopen_preview       = 3;
     prefs.gui_ask_unsaved            = TRUE;
     prefs.gui_find_wrap              = TRUE;
-    prefs.gui_webbrowser             = g_strdup("mozilla %s");
+    prefs.gui_webbrowser             = g_strdup("sensible-browser %s");
     prefs.gui_window_title           = g_strdup("");
     prefs.gui_layout_type            = layout_type_5;
     prefs.gui_layout_content_1       = layout_pane_content_plist;
